### DISCUSSION TITLE
Can I get a Sample Vault?

### DISCUSSION THREAD

**Discussion by KuiyueRO** [chrisgurney/obsidian-note-toolbar#1] [Q&A] [answered]
Thank you for the plugin you developed, I like it very much.

I want to learn some advanced usage methods. I want to see how it is implemented and to what extent it can be achieved.

: )

**Comment by chrisgurney** [comment, marked as answer]
Thank you for your feedback!

~~I'm aiming to write up some examples in the wiki section of this repo, with instructions for each command.~~ Update: I've updated [the wiki](https://github.com/chrisgurney/obsidian-note-toolbar/wiki) with detailed instructions and some examples.

Because the samples I shared in the readme are so dependent on other plugins, I think if I add a sample vault to the project, it will be for code-testing purposes. [The wiki](https://github.com/chrisgurney/obsidian-note-toolbar/wiki) I think is a better place for such examples.

Thanks again!
